packages/snaps-controllers/src/cronjob/CronjobController.ts

```diff
@@ -152,9 +152,26 @@ export class CronjobController extends BaseController<
     this._handleEventSnapUpdated = this._handleEventSnapUpdated.bind(this);
     this._handleSnapDisabledEvent = this._handleSnapDisabledEvent.bind(this);
     this._handleSnapEnabledEvent = this._handleSnapEnabledEvent.bind(this);
+
     // Subscribe to Snap events
-    /* eslint-disable @typescript-eslint/unbound-method */
+    this.#initializeEventListeners();
+
+    // Register action handlers
+    this.#initializeActionHandlers();
+
+    this.dailyCheckIn().catch((error) => {
+      /* istanbul ignore next */
```

Comment:
Added ignore statement here (and below) as this scenario is pretty difficult to replicate in our test environment. I don't think it's worth the effort to try and cover these.

Reply:
Should we really add an ignore or leave this uncovered in the coverage report ? I feel like it would be better

Reply:
I think leaving it uncovered would make sense if we ever intend on trying to cover this, otherwise it's just a nuisance to look at in coverage reports. cc: @FrederikBolding